.env.example

```diff
@@ -26,6 +26,10 @@ KAFKA_BROKER=localhost:9092
 #EMAIL
 SENDER_EMAIL=
```

Comment:
remove sender email and sender password if not used anymore.

Reply:
They are required sender email for to pass in mail options and password to give as pass key. 

Added env in yml file and also added all of them .env.example.